### PR TITLE
COVID-19 Detector

### PR DESCRIPTION
**Pull request by TEMESGEN-G25**
# Summary

- The project is a single-page application (SPA) built with React and Redux.
- Created unit tests for pure functions in Redux code.
- Created integration tests application using the React Testing Library
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The data retrieved from the API is stored in the Redux store.
- I selected an API that provides numeric data about COVID-19 global cases and then I built the web app around it. The web app has several pages:

1. The homepage lists countries that some parameters could filter. Along with the country name, it'll display some numeric values per category that come from the API. When the user clicks (or taps) on a country, the application navigates to the details page.
2. The details page is for the country's details. Retrieves data from the API to show detailed data of the confirmed cases of each region.

## Video Presentation
[Video](https://www.loom.com/share/cf5dba72edc34d0f9e89d84aaf56787b)

## Live Demo
[Link](https://covid19-globally.netlify.app/)

### CODE REVIEW

File: src/components/styles/Homepage.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+.listItem {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  padding: 10px 20px;
+}
+
+.listItem:nth-child(3n-1) {
+  background-color: #d04e79;
+}
```
Comment (elisha2kyakpo1) [inline src/components/styles/Homepage.css:55]: - [ ] [OPTIONAL] This part of the styling can be improved a bit, you can do a similar thing to the second nth-child to make it look exactly like the original.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @NatiG25, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The PR title and description looks professional 👍
- No linter errors 👍
- The readme file is well documented 👍
- You have descriptive commit messages 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/styles/Homepage.css (hunk 1 above)
Comment (TEMESGEN-G25) [reply src/components/styles/Homepage.css:55]: @elisha2kyakpo1 Thank you so much for your suggestion. But I didn't understand what you meant exactly.

**Comment by TEMESGEN-G25** [comment]
@elisha2kyakpo1 Thank you for your review :smile:.



**Note**: All optional & required changes have been implemented.

**Pull request merged**